### PR TITLE
Prediction through ANN #5

### PR DESCRIPTION
**Pull request by NeonKazuha**
- Thoroughly cleaned, preprocessed data.
- Used Artificial Neural Networks with Sequential, Dense, BatchNormalization and Dropout.
- The activation function used is relu.
- Optimizer used is adam.
- Loss used is Mean Squared Error.
- Metrics used is Mean Absolute Error.
Screenshot 2024-03-15 133141

- As both the loss and Mean absolute errors are of negligible value, this ANN works efficiently.

Screenshot 2024-03-15 132215

### CODE REVIEW

**Comment by darshbaxi** [comment]
Awesome! looks good:)
